pythonforandroid/recipes/jedi/__init__.py

```diff
@@ -1,13 +1,11 @@
-
 from pythonforandroid.recipe import PythonRecipe
 
 
 class JediRecipe(PythonRecipe):
-    # version = 'master'
     version = 'v0.9.0'
     url = 'https://github.com/davidhalter/jedi/archive/{version}.tar.gz'
 
-    depends = [('python2', 'python3crystax')]
+    depends = [('python2', 'python3crystax', 'python3')]
```

Comment:
It isn't really documented anywhere, but you can/should omit this from the dependencies in which case all python recipes will be added automatically.

Reply:
Good to know thanks!